src/store/Account.js

```diff
@@ -1195,6 +1281,42 @@ const actions = {
       })
   },
 
+  getCustodyBillingReport ({ commit, getters }, { params }) {
+    commit(types.GET_CUSTODY_BILLING_REPORT_REQUEST)
+    params = { ...params, accountId: getters.accountId }
```

Comment:
I think it is bad practice to mutate variable `params`. I suggest to create new variable for modified data